Solutions/Malware Protection Essentials/ReleaseNotes.md

```diff
@@ -0,0 +1,3 @@
+| **Version** | **Date Modified (DD-MM-YYYY)** | **Change History** |
+|-------------|--------------------------------|--------------------|
+|3.0.0        |12-21-2023                      |Initial Release     |
```

Comment:
Can check with PM, but as per me, in change history, we should be adding all the details of what we are bringing in this solution.

Reply:
This is to capture the changes history, this is fine. We'll write a blog post for solution details.